landing-page/components/Footer.tsx

```diff
@@ -0,0 +1,157 @@
+                <img className="rounded-full h-12" src="https://avatars.githubusercontent.com/u/57251443?v=4"/>
+                <div className="ml-3">
+                 <span className="text-base ml-1 mt-2">
+                   Christian
+                 </span>
+                  <div className="text-xs ml-1 text-secondary">
+                    <span>
+                      Project Management
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="bg-gray-300 rounded-3xl">
+              <div className="flex m-3">
+                <img className="rounded-full h-12" src="https://avatars.githubusercontent.com/u/13922835?v=4"/>
+                <div className="ml-3">
+                 <span className="text-base ml-1 mt-2">
+                   Felix
+                 </span>
+                  <div className="text-xs ml-1 text-secondary">
+                    <span>
+                      Project Management
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="bg-gray-300 rounded-3xl">
+              <div className="flex m-3">
+                <img className="rounded-full h-12" src="https://avatars.githubusercontent.com/u/67233923?v=4"/>
+                <div className="ml-3">
+                 <span className="text-base ml-1 mt-2">
+                   Felix
+                 </span>
+                  <div className="text-xs ml-1 text-secondary">
+                    <span>
+                      Frontend Developer
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="bg-gray-300 rounded-3xl">
+              <div className="flex m-3">
+                <img className="rounded-full h-12" src="https://avatars.githubusercontent.com/u/22842759?v=4"/>
+                <div className="ml-3">
+                 <span className="text-base ml-1 mt-2">
+                   Max
+                 </span>
+                  <div className="text-xs ml-1 text-secondary">
+                    <span>
+                      Backend Developer
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="bg-gray-300 rounded-3xl">
+              <div className="flex m-3">
+                <img className="rounded-full h-12" src="https://avatars.githubusercontent.com/u/91319622?v=4"/>
+                <div className="ml-3">
+                 <span className="text-base ml-1 mt-2">
+                   Jonas
+                 </span>
+                  <div className="text-xs ml-1 text-secondary">
+                    <span>
+                      Project Management
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
```

Comment:
this is **really** repetitive, use a custom component and use it to map over an array